### ISSUE TITLE
Manage Language Models not clearly identifying all information

### ISSUE THREAD

**Issue by accesswatch** [microsoft/vscode#280641]
Type: <b>Bug</b>

Using VSCode Insiders.

Using JAWS for Windows 2026.

1. Open VSCode and open the agent view (ctrl+alt+i).
2. Move to the model picker and press enter ... You are placed on auto. Tab to configure models.
3. You land on a search field.
4. Tab a couple of times and you enter a list. As just an interesting thought, shouldn't this be a treeview with the provider being the parent with sublings underneath?
5. Now, arrow down. You hear each of the models but none of the supplied information that is visually showing on the screen.
6. It is unclear on the user experience here. If I tab after selecting an item I get options to hide from chat and then see buttons that say Tools and Visual. These often repeat when pressing the tab key.

I do have some questions as well:

1. Should their be a checkbox here on an item or is there one and if so the checked state is not showing.
2. How does one download these models specifically if not downloaded from this tool?

Sorry for the lack of knowledge on the intended user experience on this dialog.

VS Code version: Code - Insiders 1.107.0-insider (6e0893efb3d4b50c43dcff8979c722e6019de61d, 2025-12-02T05:02:35.190Z)
OS version: Windows_NT x64 10.0.26200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i5-1340P (16 x 2189)|
|GPU Status|2d_canvas: enabled<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.69GB (2.93GB free)|
|Process Argv|--crash-reporter-id 9a7df1cb-b8fa-4069-adeb-4869054aaf64|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (43)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.5.0
vscode-intelephense-client|bme|1.16.2
npm-intellisense|chr|1.4.5
vscode-markdownlint|Dav|0.60.0
vscode-eslint|dba|3.0.21
vscode-firefox-debug|fir|2.15.0
codespaces|Git|1.18.3
copilot-chat|Git|0.34.2025120202
remotehub|Git|0.65.2025081801
vscode-github-actions|git|0.28.1
vscode-pull-request-github|Git|0.122.1
azure-dev|ms-|0.10.0
vscode-azure-github-copilot|ms-|1.0.137
vscode-azure-mcp-server|ms-|1.0.1
vscode-azureappservice|ms-|0.26.4
vscode-azurecontainerapps|ms-|0.10.0
vscode-azurefunctions|ms-|1.20.2
vscode-azureresourcegroups|ms-|0.11.7
vscode-azurestaticwebapps|ms-|0.13.2
vscode-azurestorage|ms-|0.17.1
vscode-azurevirtualmachines|ms-|0.6.10
vscode-containers|ms-|2.3.0
vscode-cosmosdb|ms-|0.30.1
vscode-docker|ms-|2.0.0
vscode-edge-devtools|ms-|2.1.10
data-workspace-vscode|ms-|0.6.3
mssql|ms-|1.37.0
sql-bindings-vscode|ms-|0.4.1
sql-database-projects-vscode|ms-|1.5.5
playwright|ms-|1.1.17
debugpy|ms-|2025.17.2025112401
python|ms-|2025.19.2025120101
vscode-pylance|ms-|2025.10.3
vscode-python-envs|ms-|1.13.13351017
remote-wsl|ms-|0.104.3
azure-repos|ms-|0.41.2025081801
powershell|ms-|2025.4.0
remote-repositories|ms-|0.43.2025081801
vscode-node-azure-pack|ms-|1.8.0
windows-ai-studio|ms-|0.26.3
chatgpt|ope|0.5.47
vscode-ai-foundry|Tea|0.12.4
context7-mcp|Ups|1.0.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
pythonvspyt551cf:31249598
nativeloc1:31118317
dwcopilot:31158714
dwoutputs:31242946
copilot_t_ci:31333650
g012b348:31231168
pythonrdcb7:31268811
pythonpcpt1:31399616
6518g693:31302842
6abeh943:31336334
envsactivate1:31349248
editstats-enabled:31346256
cloudbuttont:31366566
aihoversummaries_t:31413033
todos-1:31405333
qwen_all_req:31396049
3efgi100_wstrepl:31403338
trigger-command-fix:31379601
auto_model_enabled:31385282
use-responses-api:31390341
je187915:31401257
nes-next-cursor-1104-new-ux:31426498
3i15c187:31419923
agentic_ask_h7d75428:31406124
copilot-nes-oct-t:31413017
terminalsuggestenabled:31421703
none:31426257
cp_cls_c_966_ss:31426491
chatvisible:31417128
expandedtokens:31418009
a9239246:31426643
c0683394:31426860
decaj161:31422462
find_all_ref_in_bg_t:31420845
3fia6346:31426581
478ah919:31426749
ge8j1254_inline_auto_hint_haiku:31426887
diagnostics:31426890
b53de340:31426751
fa76a614:31426880
5j92g670_sonnet:31426787
no_request_with_suggest:31426863
rename_enabled:31426869
nes-joint-1:31426897

```

</details>

**Comment by accesswatch** [comment]
@meganrogge  Can you provide any comment on the intended experience here by chance?

**Comment by meganrogge** [comment]
This is now fixed
